Hardcore.lua

```diff
@@ -27,6 +27,7 @@ local GRIEF_WARNING_BOTH_FACTIONS = 3
 Hardcore_Settings = {
 	notify = true,
 	level_list = {},
+	debug_log = {},
```

Comment:
We would instruct mods to log off and inspect saved_vars. A list of detected attacks and other things would be here